core/src/subgraph/context/instance.rs

```diff
@@ -36,6 +37,8 @@ where
     pub fn from_manifest(
         logger: &Logger,
         manifest: SubgraphManifest<C>,
+        static_data_sources: Vec<DataSource<C>>,
```

Comment:
the terminology we use is data sources and dynamic data source, please keep this for consistency

Reply:
ty for pointing out, actually that vec is not at all needed there since manifest carries it, so i just removed it. the reasoning behind `static_data_sources` naming was that in the same code `data_sources` is used to indicate a vec containing both static and dynamic ds's